### PR TITLE
[WIP] doc: Add documentation regarding usage of the board

### PR DESCRIPTION
**Pull request by rshym**

### CODE REVIEW

File: usage.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+- [Flashing the image of the product](#flashing-the-image-of-the-product)
+- [Run on SD-card](#run-on-sd-card)
+- [Run on eMMC](#run-on-emmc)
+- [What to do on the board](#what-to-do-on-the-board)
+
+
+## What you have to have
+
+Product files:
+- `full.img` - file with product's image.
```
Comment (lorc) [inline usage.md:21]: It will be nice to describe how to get this if you don't have it.

File: usage.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+- `ipl-MACHINE.tar.bz2` - initial program loaders, OP-TEE OS and u-boot
```
Comment (lorc) [inline usage.md:22]: The same. How to obtain this?

File: usage.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+- Starter Kit Premier with Kingfisher - same as for "Starter Kit Premier (H3SK)"
+- AosBox (CCPF-SK) - https://elinux.org/R-Car/Boards/CCPF-SK
+- Salvator-X, -XS with H3 - https://xen-troops.github.io/documentation/Connect-cables-to-Salvator-X,-XS
+
+### Ethernet
+To flash the image file you need to have TFTP server on your PC and the board have to be connectged to the same network as your PC.
+You may connect the board to the network switch/router or to the second ethernet adapter on your PC.
+
+### Flashing script
+The rcar_flash script is required to flash the loaders. It is located at https://github.com/xen-troops/rcar_flash.
```
Comment (lorc) [inline usage.md:56]: It is not strictly "required". You can replace this with "used".

File: usage.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+or `bmap`
+```
+sudo bmaptool copy ./full.img <path_to_your_sd_card_like_/dev/sdX>
+```
+Setup the u-boot environment to run from SD-card by default.
+- Start serial terminal `minicom -b 115200 /dev/ttyUSB0`.
+- Turn on the board.
+- Wait few seconds until u-boot prrints "Hit any key to stop autoboot".
+- Press Enter and see "=>" u-boot prompt.
+- Set boot from SD-card as default for u-boot `setenv bootcmd run bootcmd_mmc0`
```
Comment (lorc) [inline usage.md:106]: You are assuming that U-Boot env has already `bootcmd_mmc0` defined

File: usage.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+## What to do on the board
+
+Use `root` to login to Dom0 or DomD.
+
+The mostly used program inside Dom0 is `xl`. Examples of usage:
+```
+xl list                         - list running domains
+xl console DomD                 - switch to DomD's console (press Ctrl-5 to return to Dom0)
+xl destroy DomU                 - destroy DomU
```
Comment (lorc) [inline usage.md:141]: Please, no :)

`systemctl stop domu`.

File: usage.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+xl create /etc/xen/domd.cfg -c  - create DomU and switch to it's console
+```
+
+Detailed manual for `xl` you can find at https://xenbits.xen.org/docs/unstable/man/xl.1.html
+
+The DomD contains quite regular Linux so you can use lots of regular Linux tools.
+
+Also, pay attention that DomD is home for backend drivers for guest domains like DomU or DomA. This means that DomU/DomA is working with real hardware using drivers in DomD. And if you stop network in DomD, DomU/DomA will be isolated from the world as well.
+
+If you want to return to Dom0 from any other domain - use `Ctrl-5`.
```
Comment (lorc) [inline usage.md:151]: Not `ctrl+]`?

**Comment by rshym** [comment]
Significant rework is required, so I'm closing this hanging PR.

**Pull request closed without merging**